libs/core/src/lib/object-marker/object-marker.component.ts

```diff
@@ -0,0 +1,66 @@
+import {
+    Component,
+    OnInit,
+    ViewEncapsulation,
+    ChangeDetectionStrategy,
+    Input,
+    OnChanges,
+    ElementRef
+} from '@angular/core';
+import { CssClassBuilder, applyCssClass } from '../utils/public_api';
+
+@Component({
+    // tslint:disable-next-line:component-selector
+    selector: '[fd-object-marker]',
+    template: ` <i class="fd-object-marker__icon" [ngClass]="' sap-icon--' + glyph" *ngIf="glyph"></i>
+        <span *ngIf="label" class="fd-object-marker__text">{{ label }}</span>
+        <ng-content></ng-content>`,
+    styleUrls: ['./object-marker.component.scss'],
+    encapsulation: ViewEncapsulation.None,
+    changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class ObjectMarkerComponent implements OnChanges, OnInit, CssClassBuilder {
+    /** User's custom classes */
+    @Input()
+    class: string;
+    /**
+     * Glyph (icon) of the Object Status.
+     */
+    @Input()
+    glyph: string;
```

Comment:
Just one thing regarding comments. Could you also add 1 line of space between them and change:
```    
/**
     * Glyph (icon) of the Object Status.
 */
```
To:
`/** Glyph (icon) of the Object Marker */`